subgraph/origin/schema.graphql

```diff
@@ -0,0 +1,33 @@
+type DeclaredDepositIntent @entity {
+  id: ID!
+  tokenAddress: Bytes! # address
+  amount: BigInt! # uint256
+  nonce: BigInt!  # uint256
+  beneficiary: Bytes! # address
+  feeGasPrice: BigInt! # uint256
+  feeGasLimit: BigInt! # uint256
+  depositor: Bytes! # address
+  blockNumber: BigInt! # uint256
+  blockHash: Bytes! # bytes32
```

Comment:
messageHash is missing here

Reply:
`messageHash` added in `DeclaredDepositIntent` entity.